src/Diagnostics.ModelsAndUtils/Models/ResponseExtensions/SummaryCard.cs

```diff
@@ -0,0 +1,193 @@
+        ///}
+        /// </code>
+        /// </example>
+    public static DiagnosticData AddSummaryCards(this Response response,List<SummaryCard> summaryCards)
+        {
+            if (summaryCards == null || !summaryCards.Any())
+            {
+                throw new ArgumentException("Paramter List<SummaryCard> is null or contains no elements.");
+            }
+
+            var table = new DataTable();
+            table.Columns.Add("Status",typeof(string));
+            table.Columns.Add("Title", typeof(string));
+            table.Columns.Add("Message",typeof(string));
+            table.Columns.Add("Description", typeof(string));
+            table.Columns.Add("Link", typeof(string));
+            table.Columns.Add("ActionType", typeof(string));
+
+            foreach (var summaryCard in summaryCards)
+            {
+                DataRow nr = table.NewRow();
+                nr["Status"] = summaryCard.Status;
+                nr["Title"] = summaryCard.Title;
+                nr["Message"] = summaryCard.Message;
+                nr["Description"] = summaryCard.Description;
+                nr["Link"] = summaryCard.Link;
+                nr["ActionType"] = summaryCard.ActionType;
+                table.Rows.Add(nr);
+                //table.Rows.Add(summaryCard.Status,summaryCard.Title,summaryCard.Message,summaryCard.Description);
+            }
+
+            var diagData = new DiagnosticData()
+            {
+                Table = table,
+                RenderingProperties = new Rendering(RenderingType.SummaryCard)
+            };
+            response.Dataset.Add(diagData);
+            return diagData;
+        }
+
+        /// <summary>
+        /// Add a single summary card
+        /// </summary>
+        /// <param name="response">Response</param>
+        /// <param name="status">SummaryCardStatus</param>
+        /// <param name="title">Title</param>
+        /// <param name="message">Message</param>
+        /// <param name="description">Description</param>
+        /// <param name="link">Link</param>
+        /// <param name="actionType">ActionType</param>
+        /// <returns></returns>
+        /// <example>
+        /// This sample code shows how to use <see cref="AddSummaryCard"> method.
+        /// <code>
+        /// /// public async static Task<![CDATA[<Response>]]> Run(DataProviders dp, OperationContext cxt, Response res)
+        ///{
+        ///     res.AddSummaryCard(
+        ///         SummaryCardStatus.Info
+        ///         "Request and Errors",
+        ///         "25",
+        ///         "HTTP errors",
+        ///         "http4xx",
+        ///         SummaryCardActionType.Detector);
+        ///}     
+        /// </code>
+        /// </example>
+        public static DiagnosticData AddSummaryCard(this Response response, SummaryCardStatus status,string title,string message,string description,string link,SummaryCardActionType actionType = SummaryCardActionType.Detector)
```

Comment:
I think its better to not default the action type and let user specify it since they anyway would have to specify the id of the detector.  It makes the detector code more readable .

For ex: 
res.AddSummaryCard(... , ...., SummaryCardActionType.Detector, "bestpractices"); 
is more readable than
res.AddSummaryCard(... , ...., "bestpractices");